src/model/game/GameManager.ts

```diff
@@ -586,26 +586,26 @@ export default class GameManager {
         return this._activeGame;
     }
 
-    static get gameList(): Game[] {
-        return [...this._gameList];
-    }
-
     static set activeGame(game: Game) {
         this._activeGame = game;
     }
 
+    // Used for loading game specific settings before game is selected.
+    static get defaultGame(): Game {
+        return this._gameList.find(value => value.internalFolderName === "RiskOfRain2")!;
+    }
```

Comment:
Why not just hardcode the pointer to this game instance? Why do we need to enumerate any lists? We know the exact value this should point to at build time, there's no need to leave room for uncertainty

Reply:
I'm not entirely sure why this has been implemented like this earlier. Took a quick peek now at the git history but that didn't provide much information. I can change the implementation if we think that's safe to do, but that'll have to wait until after my vacation.

Reply:
I'm fairly sure it was done this way because of uncertainty towards moving to a cloud based game list. If the game were to be removed then we'd return undefined rather than erroring on the `.internalFolderName` because `_gameList.RiskOfRain2` wouldn't exist in the result